docs/content/preview/yugabyte-voyager/migrate/live-fall-back.md

```diff
@@ -1320,6 +1320,18 @@ Note that after you end the migration, you will _not_ be able to continue furthe
 
 Refer to [end migration](../../reference/end-migration/) for more details on the arguments.
 
+{{< warning title="Deleting the ybvoyager user" >}}
+
+After migration, all the migrated objects (tables, views, and so on) are owned by the `ybvoyager` user. You should transfer the ownership of the objects to some other user (for example, `yugabyte`) and then delete the `ybvoyager` user. Example steps to delete the user are:
```

Comment:
at this point, are they signed in as ybvoyager? I don't see any other user created in previous instructions. If so, presumably they need to create another superuser, or switch to the yugabyte user, and then delete ybvoyager. What typically is done at this point?